.prettierignore

```diff
@@ -0,0 +1,6 @@
+# Add files here to ignore them from prettier formatting
+
+/dist
+/coverage
+package-lock.json
+node_modules
```

Comment:
insert newline at the end of file

Reply:
newline added

related commit - a2757e71aa5bff388d5fbedca9e60fb31ed95a15